README.md

```diff
@@ -98,6 +98,65 @@ Here `ClangCompiler` can be replaced with one of `"Compiler" /. CCompilers[Full]
 
 A less frequently updated version is available through the Wolfram's public paclet server and can be installed by running `PacletInstall["SetReplace"]`.
 
+# Symbols and Functions
+
+## SetReplace, SetReplaceList, SetReplaceAll, SetReplaceFixedPoint, SetReplaceFixedPointList
+
+`SetReplace` (and related `SetReplaceList`, `SetReplaceAll`, `SetReplaceFixedPoint` and `SetReplaceFixedPointList`) are the functions the package is named after. They are quite simple, don't have a lot of options, and simply perform replacement operations either one-at-a-time (as in the case of `SetReplace`), to all non-overlapping subsets (`SetReplaceAll`), or until no more matches can be made (`SetReplaceFixedPoint`). A suffix `*List` implies the function will return a set after each replacement instead of just the final result.
```

Comment:
Are you going to not say in the beginning what is the argument structure for any SetReplace functions?

Reply:
Well, I first describe what they do, and then how to use them. The argument structure is basically what the whole section is about after the `WolframModel` mention in the second paragraph. Do you think I should riffle the first two paragraphs with the rest?

Reply:
I am not sure what is the best way to do this. I think giving the examples first is fine but it is probably overall argument structure at the end. Like say I am somebody who doesn't know WL very well, actually writing SetReplace[list, rule, options] might help me visualize how to do it.

Reply:
Oh, I see what you are saying. I added `SetReplace[set, rule]` pseudo-input at the beginning. The same for `WolframModel`. I did not do that for other functions as they only have one (or two similar like `HypergraphUnifications`) argument.